### PR TITLE
Feat: templates and copy for California State Transit

### PR DESCRIPTION
**Pull request by thekaveman**
This is our sample transit agency used for testing and demos.

## Templates and files

- [x] Agency index template
- [x] Help template for CST Agency Card
- [x] Eligibility index template
- [x] Eligibility selection label template for CST Agency Card
- [x] Eligibility `idcardcheck` media item template for CST Agency Card
- [x] Eligibility start template for CST Agency Card
- [x] Eligibility unverified template for CST Agency Card
- [x] Enrollment success template
- [x] Enrollment success template for CST Agency Card
- [x] `cst-lg.svg` added to `benefits/static/img/agencies`
- [x] `cst-sm.svg` added to `benefits/static/img/agencies`

Part of #2189

### CODE REVIEW

File: benefits/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -824,16 +912,12 @@ msgstr ""
...
-"You were not charged anything today. Thank you for using Cal-ITP Benefits!"
-msgstr "No se le cobró nada hoy. ¡Gracias por usar Cal-ITP Benefits!"
-
 msgid ""
 "You were not charged anything today. When boarding an MST bus, tap this card "
 "and you will be charged a reduced fare. You will need to re-enroll if you "
 "choose to change the card you use to pay for transit service."
 msgstr ""
-"No se le cobró nada hoy. Al abordar un autobús de CST, toque el lector con "
+"No se le cobró nada hoy. Al abordar un autobús de MST, toque el lector con "
```
Comment (thekaveman) [inline benefits/locale/es/LC_MESSAGES/django.po:920]: Small bug in our existing translation -- we had the `CST` slug in the `MST` copy!

**Review by angela-tran — CHANGES_REQUESTED** [review]
I think the templates look good - just some filenames typos I noticed

File: benefits/eligibility/templates/eligibility/includes/selection-label--cst-agecy-card.html
[the remark this answers is not in the document]
Comment (thekaveman) [reply benefits/eligibility/templates/eligibility/includes/selection-label--cst-agecy-card.html:1]: Thank you for catching this!

File: benefits/eligibility/templates/eligibility/start--cst-agecy-card.html
[the remark this answers is not in the document]
Comment (thekaveman) [reply benefits/eligibility/templates/eligibility/start--cst-agecy-card.html:1]: Thank you for catching this!

**Review by angela-tran — APPROVED** [review]

**Pull request merged**